<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>20대 독서모임</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="member_tap">
    <attribute name="title">
     <string>맴버 관리</string>
    </attribute>
    <widget class="QPushButton" name="adding_button">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>30</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>맴버 추가</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removing_button">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>160</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>선택 맴버 제거</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="member_table">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>600</width>
       <height>550</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>이름</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>성별</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>값</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>선택</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="deciding_button">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>480</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>선택 맴버 참석자 선정</string>
     </property>
    </widget>
    <widget class="QPushButton" name="finding_button">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>330</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>맴버 찾기</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_box">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>289</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="group_tap">
    <attribute name="title">
     <string>그룹 배정</string>
    </attribute>
    <widget class="QTableWidget" name="group_table">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>601</width>
       <height>551</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <column>
      <property name="text">
       <string>그룹</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>이름</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>성별</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>값</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>선택</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="grouping_button">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>150</y>
       <width>151</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>그룹 배정</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reset_button">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>280</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>초기화</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
